### PR TITLE
Feat: Add feature/1

### LINKED ISSUE #1: Feature 1
## 목적
프로젝트 간단 요약을 작성하기
## 작업내용
프로젝트 간단 요약을 했습니다.

### PR DESCRIPTION
**Pull request by kyungkyuBae**
## 개요
Close #1

### CODE REVIEW

**Comment by LeHiHo** [comment]
close #1

**Comment by humnyenye** [comment]
good

**Comment by suehub** [comment]
good!!!

File: readme.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 1. 프로젝트 간단 요약
+프로젝트를 간단 요약 해보았습니다.
```
Comment (noSPkeepgoing) [inline readme.md:2]: 멋져요

**Pull request merged**